continuous_integration/install.cmd

```diff
@@ -18,7 +18,7 @@ python --version
 pip --version
 
 @rem Install dependencies with either conda or pip.
-if "%PACKAGER%" == "conda" (%CONDA_INSTALL% numpy=1.15 scipy pytest cython)
+if "%PACKAGER%" == "conda" (%CONDA_INSTALL% numpy scipy pytest cython)
```

Comment:
I also unpinned the numpy version here. It was probably added at some point for a good reason but there's no explanation and it works without it.